manifests/deis-router-rc.yaml

```diff
@@ -23,3 +23,7 @@ spec:
         ports:
         - containerPort: 80
           hostPort: 80
+        - containerPort: 443
+          hostPort: 443
+        - containerPort: 2222
```

Comment:
What I was aiming for here is can `builder` append to the `host -> port` mapping from its manifest / chart so that the `router` doesn't have to? Or are there limitations on how much `rc`s can affect one another like that?

This isn't a `builder` specific question per se and more if X arbitrary chart can add to the mapping to expose different ports on the router for various random needs.